### PR TITLE
Dummy Table

### PR DESCRIPTION
**Pull request by Mahnoor-Faisal0303**

### CODE REVIEW

File: src/Components/CustomTable.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import TableRow from '@mui/material/TableRow';
+import CustomTableProps from './ItemInterface';
+
+const CustomTable: React.FC<CustomTableProps> = ({ data }) => {
+  return (
+    <TableContainer id="custom-table-container">
+      <Table id="custom-table">
+        <TableHead>
+          <TableRow>
+            <TableCell id='fontStyle'>Dessert (100g serving)</TableCell>
```
Comment (saad277) [inline src/Components/CustomTable.tsx]: can we pass these from props as well ? 
as tables can have different headers

File: src/Components/CustomTable.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            <TableCell id='fontStyle'>Calories</TableCell>
+            <TableCell id='fontStyle'>Fat (g)</TableCell>
+            <TableCell id='fontStyle'>Carbs</TableCell>
+            <TableCell id='fontStyle'>Protein</TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {data.map((item) => (
+            <TableRow key={item.id} id="custom-table-row">
```
Comment (saad277) [inline src/Components/CustomTable.tsx]: add unique id for each element

File: src/Components/Data.tsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export const data = [
```
Comment (saad277) [inline src/Components/Data.tsx]: give a type to this data

File: src/App.css [hunk 4]
```diff
@@ -1,54 +1,18 @@
...
-  padding: 2rem;
-  text-align: center;
-}
-/*
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
-  transition: filter 300ms;
+body {
```
Comment (saad277) [inline src/App.css:1]: nver style a body always keep 
margin:0;
padding:0;
background color only

File: src/App.css [hunk 5]
```diff
@@ -1,54 +1,18 @@
...
+  margin: 0;
 }
-.logo:hover {
-  filter: drop-shadow(0 0 2em #646cffaa);
+#custom-table {
+  min-width: 850px; 
 }
-.logo.react:hover {
-  filter: drop-shadow(0 0 2em #61dafbaa);
+#fontStyle{
```
Comment (saad277) [inline src/App.css]: use material classes for these

File: src/App.css [hunk 6]
```diff
@@ -1,54 +1,18 @@
...
-
-@media (prefers-reduced-motion: no-preference) {
-  a:nth-of-type(2) .logo {
-    animation: logo-spin infinite 20s linear;
-  }
-}
-
-.card {
-  padding: 2em;
+#textStyle{
```
Comment (saad277) [inline src/App.css]: use material classes for these

File: src/Components/CustomTable.tsx [hunk 7]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import { TableHeaderProps } from './ItemInterface';
+
+const CustomTable: React.FC<TableHeaderProps> = ({ data, headers }) => {
```
Comment (saad277) [inline src/Components/CustomTable.tsx]: props should be table props which have 
{
headers,
data
}

File: src/App.css (hunk 4 above)
Comment (Mahnoor-Faisal0303) [reply src/App.css:1]: ok

**Pull request merged**